go/porcelain/deploy.go

```diff
@@ -460,10 +460,11 @@ func walk(dir string, observer DeployObserver) (*deployFiles, error) {
 		}
 
 		if !info.IsDir() && info.Mode().IsRegular() {
-			rel, err := filepath.Rel(dir, path)
+			osRel, err := filepath.Rel(dir, path)
 			if err != nil {
 				return err
 			}
+			rel := cleanFileName(osRel)
```

Comment:
you could use `filepath.Clean` https://golang.org/pkg/path/filepath/#Clean

Reply:
I'm going to change the function name, it's misleading.

Reply:
@brycekahle 
```
Finally, any occurrences of slash are replaced by Separator.
```

Isn't that what we want?

Reply:
No, because on windows, separator = `\`

Reply:
That means that for any windows machine, this will force `/` not `\`? If you guys say that is right then ok! You've got more context than me.

Reply:
This is precisely the bug we are trying to fix! https://github.com/netlify/netlifyctl/issues/109